src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/Pillar2SubmissionPage.scala

```diff
@@ -21,7 +21,7 @@ import org.scalatest.concurrent.Eventually
 import org.scalatest.matchers.should.Matchers
 import uk.gov.hmrc.test.ui.cucumber.PageObject
 import uk.gov.hmrc.test.ui.driver.BrowserDriver
-import uk.gov.hmrc.test.ui.pages.{P2SubBtnAccountingPage, P2SubBtnAnnualRevenuePage, P2SubBtnDomesticOrMnePage, P2SubBtnFutureAnnualRevenuePage, P2SubBtnStartPage, P2UkTaxReturnPage}
+import uk.gov.hmrc.test.ui.pages.{AccountsSummaryPage,P2SubBtnAccountingPage, P2SubBtnAnnualRevenuePage, P2SubBtnDomesticOrMnePage, P2SubBtnFutureAnnualRevenuePage, P2SubBtnStartPage, P2UkTaxReturnPage}
```

Comment:
You should probably change this to an underscore rather than have 7 imports, especially with the new code linter.